sdk/communication/communication-sms/samples/javascript/sendSms.js

```diff
@@ -21,21 +21,21 @@ async function main() {
 
   const client = new SmsClient(connectionString);
 
-  // Your E.164 formatted phone number used to send SMS
-  const from = "<phone number>";
-
-  // The list of E.164 formatted phone numbers to which message is being send
-  const to = ["<phone number>"];
-
-  // The message being sent
-  const message = "Hey!";
-
-  console.log("-- Sending SMS --");
-
   // Send SMS message
-  await client.send({ from, to, message }, { enableDeliveryReport: false });
-
-  console.log("Message sent!");
+  const sendResults = await client.send(
+    {
+      from: "<phone number>", // Your E.164 formatted phone number used to send SMS
+      to: ["<phone number>", "<phone number>", "<phone number>"], // The list of E.164 formatted phone numbers to which message is being send
+      message: "Hello World via SMS!" // The message being sent
+    },
+    {
+      enableDeliveryReport: true,
+      tag: "customTag"
+    }
+  );
+  for (const sendResult of sendResults) {
+    console.log("result: ", sendResult);
+  }
```

Comment:
We got this comment from the review that samples need to be a little more detailed.. so for Python I liked that we did a retry for all failures and also ignored the 400s and did not retry them as they will fail again..